docs/documentation/schedule/components/pathways.md

```diff
@@ -1,35 +1,35 @@
-#  Pathways Component
-The Pathways component contains features that allow the modeling of in-station, helping users navigate transit stations from beginning to end and across multiple levels. Some of these features make it possible to communicate the physical characteristics of each individual section of a path, estimate the time required to navigate them and to match this navigation information with the real-world wayfinding systems used in stations.
+#  Pathways
+Pathways contains features that allow to model large transit stations, helping riders navigate from station entrances and exists to the location where they board or disambark from a transit vehicle. Some of these features make it possible to communicate a path's physical characteristics and estimated navigation time, and real-world wayfinding systems employed in stations.
 
 ## Location types
 
 <div class="grid" markdown>
 
-This feature is used to classify relevant places within transit stations such as exits/entrances and boarding areas. This can help users by making it easier to identify the right entrance, exit or platform.
+Location types are used to classify key areas within transit stations such as exits/entrances, nodes or boarding areas, as well as their relationship. Location types serve as a fundation for modeling transit stations using Pathways.
 
 | Files associated      | [stops.txt](/schedule/reference/#stopstxt)                       |
 |-----------------------|---------------------------------|
 | **Fields associated** | location_type<br>parent_station |
 
 </div>
 
-## Pathways basic information
+## Pathways foundations
```

Comment:
I'm not completely sold on the title, but it's a fair option and I've had a hard time finding the right word for this feature, so let's go with it.

Reply:
Pathway connections?

Reply:
Changed to pathway connections